src/screens/Swap/manifests.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import { useMemo } from "react";
+
+import ParaswapIcon from "../../icons/swap/Paraswap";
+
+const useManifests = () =>
+  useMemo(() => {
+    const paraswapUrl = new URL(
```

Comment:
should these part be at least on live-common side?
- i understand it's not a blocker for MVP.
- but let's make sure to add a TODO comment for it.

Reply:
I don't think a comment is mandatory here, since implementing manifest will be the next big topic of the V1 version, this whole part of the code will disappear, and would go to LLC anyway.